src/test/java/org/prebid/server/bidder/advangelists/AdvangelistsBidderTest.java

```diff
@@ -0,0 +1,462 @@
+package org.prebid.server.bidder.advangelists;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.iab.openrtb.request.App;
+import com.iab.openrtb.request.Banner;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Format;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.request.Native;
+import com.iab.openrtb.request.Publisher;
+import com.iab.openrtb.request.Site;
+import com.iab.openrtb.request.Video;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import io.vertx.core.http.HttpMethod;
+import org.junit.Before;
+import org.junit.Test;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpCall;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.advangelists.ExtImpAdvangelists;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonList;
+import static java.util.function.Function.identity;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.banner;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.video;
+
+public class AdvangelistsBidderTest extends VertxTest {
+
+    private static final String ENDPOINT_URL = "http://test/get?pubid=";
+
+    private AdvangelistsBidder advangelistsBidder;
+
+    @Before
+    public void setUp() {
+        advangelistsBidder = new AdvangelistsBidder(ENDPOINT_URL);
+    }
+
+    @Test
+    public void creationShouldFailOnInvalidEndpointUrl() {
+        assertThatIllegalArgumentException().isThrownBy(() -> new AdvangelistsBidder("invalid_url"));
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnErrorWhenImpExtCouldNotBeParsed() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(singletonList(Imp.builder()
+                        .ext(mapper.valueToTree(ExtPrebid.of(null, mapper.createArrayNode())))
+                        .build()))
+                .id("request_id")
```

Comment:
After you've generated files, please make sure the code is fine, like this one - this request field is not being checked, therefore it is unnecessary. I'll look into the generator to clean up such cases, but please, always double check the code.

Check other occurrences again, thanks!